packages/base/src/widget.ts

```diff
@@ -689,9 +689,9 @@ class WidgetView extends NativeView<WidgetModel> {
     /**
      * Create and promise that resolves to a child view of a given model
      */
-    create_child_view(child_model: WidgetModel, options = {}): Promise<DOMWidgetView> {
+    create_child_view<VT extends DOMWidgetView = DOMWidgetView>(child_model: WidgetModel, options = {}): Promise<VT> {
```

Comment:
I think this should probably have the same overrides as the create_view? Not all children views will be dom widget views.